### PR TITLE
Update instructions for running operator locally

### LINKED ISSUE #71: "Running manually" instructions are incomplete
The current "Running manually against a test cluster" instructions are incomplete; you need to set (at least) `KUBERNETES_SERVICE_PORT` and `KUBERNETES_SERVICE_HOST` in the environment as well in order for the sdn-controller daemonset to function correctly. (Possibly others?)

### PR DESCRIPTION
**Pull request by danwinship**
Fixes #71

### CODE REVIEW

**Comment by danwinship** [comment]
(updated for `openshift-cluster-network-operator` -> `openshift-network-operator` namespace rename)

File: README.md [hunk 1]
```diff
@@ -213,18 +213,22 @@ _output/linux/amd64/cluster-network-renderer --config sample-config.yaml --out o
...
+- `NODE_IMAGE` and `HYPERSHIFT_IMAGE` - These are normally set in the operator's environment by the Cluster Version Operator, pointing to the correct versions of the dependent images.
+- `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT` - The Cluster Network Operator needs to provide the SDN controller pods with the host network address of the Kubernetes apiserver (since the `172.30.0.1` address may not be functioning when the SDN controller starts, and it is currently not possible for the SDN controller to get this information from its node in the way that the SDN node pods do).
+
+You can determine the correct values of these environment variables by inspecting the (working) cluster before you replace the operator:
 ```sh
-oc get -n openshift-network-operator daemonset cluster-network-operator -ojsonpath='{range .spec.template.spec.containers[0].env[?(@.value)]}{.name}{"="}{.value}{" "}' | tee images
+oc get -n openshift-network-operator daemonset cluster-network-operator -ojsonpath='{range .spec.template.spec.containers[0].env[?(@.value)]}{.name}{"="}{.value}{"\n"}' | tee env.sh
```
Comment (pecameron) [inline README.md]: Isn't this now a deployment?

Comment (danwinship) [reply README.md]: good catch!

**Pull request merged**